src/runtime/vdom/vdom-render.ts

```diff
@@ -111,6 +111,19 @@ const createElm = (oldParentVNode: d.VNode, newParentVNode: d.VNode, childIndex:
       elm.classList.add((elm['s-si'] = scopeId));
     }
 
+    if (BUILD.shadowDom || BUILD.scoped) {
```

Comment:
For my own understanding, why do we need both `shadowDom` and `scoped` here? Could we get by with just
```suggestion
    if (BUILD.scoped) {
```

Reply:
makes perfectly sense! done